### PR TITLE
server: fix openapi crashes

### PR DESCRIPTION
**Pull request by matildepark**
`npx nolita@experimental` is working on a Docker container, so this seems to work.

Still, not sure how I feel about the solution. You need to anticipate how a zod object will be used *elsewhere* and then change the import. Extending our home `z` with `extendZodWithOpenApi` from `@hono/zod-openapi@0.14.9` would be preferred, at the earliest possible z call we do in the stack, but it causes crashes all over the places when you do that...it could be due to them [bumping this dependency up a few major versions](https://github.com/honojs/middleware/blob/main/packages/zod-openapi/CHANGELOG.md#0145).

My actual recommendation is to rip out every time we use `.openapi()` and then reimplement it with `0.14.9` from scratch.

### CODE REVIEW

**Review by AlephNotation — APPROVED** [review]

**Comment by matildepark** [comment]
From Discord:

> tl;dr: this one actually works on prod https://github.com/hdresearch/nolita/pull/115, the other one (https://github.com/hdresearch/nolita/pull/117) will build and at least run but can't start a browser session via server because it can't parse incoming json
